src/chimera/controllers/imageserver/imagerequest.py

```diff
@@ -27,8 +27,8 @@ def __init__(self, **kwargs):
                     'window': None,
                     'bitpix': Bitpix.uint16,
                     'filename': '$DATE-$TIME',
-                    'compress': True,
-                    'compress_format': "BZ2",
+                    'compress': False,
```

Comment:
If we can pass compress: "NO", can't see the point of having a boolean flag for it as well, looks redundant. Why not simply 'compress' begin an Enum (NO, BZ2, GZIP, ZIP)?